drupal-issue-tags.js

```diff
@@ -0,0 +1,53 @@
+(async () => {
+  const src = chrome.runtime.getURL("common.js");
+  const { utils } = await import(src);
+
+  var url = document.URL;
+  const regex1 = /https:\/\/www\.drupal\.org\/project\/.*\/issues\/.*/g;
+  const regex2 = /https:\/\/www\.drupal\.org\/node\/add\/.*/g;
+  if (url.match(regex1) || url.match(regex2)) {
+    const assignedInput = document.getElementById("edit-taxonomy-vocabulary-9-und");
+    const assignedText = assignedInput.value;
+    const existingTags = assignedText.split(", ");
+
+    const issueTagsDiv = document.createElement("div");
+    issueTagsDiv.className = "issueTagsDiv";
+    let commonIssueTags = [
+      'Needs tests',
+      'Needs issue summary update',
+      'Accessibility',
+      'core-mvp',
+    ]
+    commonIssueTags.forEach((tag)=> {
+      const tagDiv = document.createElement("button");
+      tagDiv.className = "tagDiv";
+      tagDiv.innerText = `${tag}`;
+      tagDiv.id = `${tag}`;
+      if(existingTags.includes(`${tag}`)){
+        document.getElementById(`${tag}`).setAttribute("disabled", "disabled");
```

Comment:
```suggestion
        tagDiv.setAttribute("disabled", "disabled");
```
when the page is first being loaded if the tag already in the input this will cause an error because `document.getElementById(`${tag}`)` won't exist because `tagDiv` has not be added to the page yet.

see https://www.drupal.org/project/automatic_updates/issues/3318587
as an example of a issue which already has some of these tags.